src/map/script.c

```diff
@@ -9013,6 +9013,32 @@ BUILDIN(getgmlevel)
 	return true;
 }
 
+/// set the group ID of the player.
+/// setgroupid( <new group id>,{<account ID> or <Player Name>} )
+/// return 1 on success, 0 if failed.
+BUILDIN(setgroupid) {
+	TBL_PC* sd;
```

Comment:
https://github.com/HerculesWS/Hercules/commit/8382ee1cea443ac62a16c21c750423f3a2a624af
oops ... disregard about this
but when Haru review my commit, he didn't change my `TBL_PC` into `struct map_session_data`